tcms/xmlrpc/utils.py

```diff
@@ -34,10 +32,10 @@ def pre_check_product(values):
         if not product_str:
             raise ValueError('Got empty product name.')
         return Product.objects.get(name=product_str)
-    elif type(product_str) == int:  # to avoid breaking tests with bool(int)
+    # to avoid breaking tests with bool(int)
```

Comment:
This comment is now invalid so you have to remove it. While at it you can also fix the other comments which were meant as FYI only.